### PR TITLE
DA 678 Add Phone Number

### PR DESCRIPTION
**Pull request by ctsstc**
## Changes
- Add `env.example` template
- Update README with additional steps for set up
- Add phone number toggle to popup template config
- Add background image to test page
- Fix modal centering issues; especially upon resizing
- Fix undefined css when option wasn't present
- Add phone number & use phone number to api

## Stories
- https://teamg5.atlassian.net/browse/DA-678
- https://teamg5.atlassian.net/browse/DA-679

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
+Make sure you have the correct version of ruby defined in `Gemfile`. If you use something like `rvm` it will notify you to install the required version if you do not have it.
 
-* RUBY 2.6.5
+You will need bundler v2 or greater. If you do not have it run `gem install bundler`
 
-* System dependencies
+```bash
+bundle
+yarn install --check-files
+bin/rails db:create db:migrate
```
Comment (levibrown) [inline README.md]: small point, but normally you would use `bundle exec` and not bin/rails

**Comment by levibrown** [comment]
not sure what going on with codefresh, but will you look into that too?

**Comment by ctsstc** [comment]
Don took a deep dig yesterday and had no luck getting it running:

- https://getg5.slack.com/archives/CQTQ9N29Y/p1582737752015900
- https://getg5.slack.com/archives/CQTQ9N29Y/p1582746413030900

I'll try adding a github action real quick and we'll see if that works 🤞

**Comment by ctsstc** [comment]
Fixed up CI on master and merged master into this branch 🎉 

Dropped codefresh magic and went with github actions 👌

**Pull request merged**